commands/channel.go

```diff
@@ -345,32 +350,35 @@ func makeChannelPrivateCmdF(c client.Client, cmd *cobra.Command, args []string)
 }
 
 func renameChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	var newDisplayName, newChannelName string
+	existingTeamChannel := args[0]
+	newChannelName, _ := cmd.Flags().GetString("name")
+	newDisplayName, _ := cmd.Flags().GetString("display_name")
 
-	if len(args) < 2 {
-		return errors.New("Not enough arguments.")
+	// Both Display and name flags are not entered
+	if newDisplayName == "" && newChannelName == "" {
+		return errors.New("Require atleast one flag to rename team, either 'name' or 'display_name'")
 	}
 
-	channel := getChannelFromChannelArg(c, args[0])
+	channel := getChannelFromChannelArg(c, existingTeamChannel)
 	if channel == nil {
-		return errors.New("Unable to find channel '" + args[0] + "'")
+		return errors.New("Unable to find channel '" + existingTeamChannel + "'")
```

Comment:
I would suggest a slight modification to be more accurate: "Unable to find channel from ...". Because `existingTeamChannel` is not just the channel name.